### PR TITLE
Feature/fixes

### PR DESCRIPTION
**Pull request by axelniklasson**
* Meta tags for scrapers

* Render images in case view

* Lint and minor fixes

### CODE REVIEW

**Comment by axelniklasson** [comment]
Deployed on [staging](https://staging.inneva.se)

File: src/containers/Case.js [hunk 1]
```diff
@@ -63,6 +65,7 @@ class Case extends Component {
     this.state.client.getCaseBySlug(this.props.match.params.slug)
       .then(data => {
         this.setState({ caseLoading: false, data })
+        console.log(data)
```
Comment (forsstromerik) [inline src/containers/Case.js]: Remove logging

**Review by eldherik — CHANGES_REQUESTED** [review]
Looks good. Just one log to remove :)

**Comment by axelniklasson** [comment]
That was one fast review! Thanks mate, merging this one now.

**Pull request merged**